docs/production.md

```diff
@@ -1,72 +1,41 @@
 Production Environments
 =======================
 
-This page describes creating and managing production inventory files.
-For details on deploying a production environment, see the [deployment page](/deployment).
+This page describes creating and managing production environments.
 
 ### Directory overview
 
-Production environments live in the `inventories/` folder.
+Environments live in the `environments/` folder.
 
-To add a new production environment you can follow the steps below, replacing `myproject` with your project name.
+To add a new environment you can follow the steps below, replacing `your_environment` with your environment name.
 
 
-Dimagi-managed production environments are maintained in the private `inventories/dimagi/` submodule.
-Dimagi developers should see [the dimagi page](/dimagi) for details on working with this submodule
-in Dimagi-managed environments.
-
-### Initialize Inventory Folder
+### Initialize environment directory
 
 First create a new inventory folder for your environment.
 This is where your project-specific configuration will live. 
 
 ```bash
-cp -r inventories/example inventories/myproject
+cp -r environments/example environments/your_environment
 ```
 ### Update Inventory Files
 
-Edit the `hosts.yml` and `vars.yml` files with your project-specific changes.
+Edit the `inventory` and `vault.yml` files with your project-specific details.
 
 
 ### Ansible Vault
 
 Production environments should use [Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html) to manage secrets.
 That page has lots of details about editing and using files with Vault.
 
-The example environment includes a vault file which you can edit using:
+The example environment includes a vault file which you can edit. When you're done adding the necessary secrets, you must encrypt the vault file. It will ask you for a password when you run the encrypt command:
 
 ```python
-ansible-vault edit ./inventories/example/group_vars/commcare_analytics/vault.yml
-```
-
-And entering the password `secret`.
-You should remove this file and create a new one for your environment following the instructions below.
-
-### Initial Vault Setup
-
-The following one-time setup is used to generate keys / files for Ansible Vault.
-
-#### Generate Vault Key
-
-```bash
-openssl rand -base64 2048 > ~/myproject-ansible-vault
-```
-
-#### Create Vault Vars File
-```bash
-ansible-vault create --vault-password-file ~/myproject-ansible-vault ./inventories/myproject/group_vars/commcare_analytics/vault.yml
+ansible-vault encrypt vault.yml
 ```
 
-Add your secrets here. E.g.
-
-```
-# My Project Vault File
-
-vault_default_db_password: SuperSecret
-vault_django_secret_key: als0_SEcr3t
-```
+You can generate a good random key from a command line:
```

Comment:
Although you didn't write the Python snippet below, there is an easier way to get a good random key:

    $ openssl rand -base64 48

Reply:
Great suggestion, thanks!